web-registry/src/pages/Dashboard/MyEcocredits/hooks/useCreditSendSubmit.tsx

```diff
@@ -69,11 +83,32 @@ const useCreditSendSubmit = ({
         memo: note,
       };
 
-      const onError = (): void => {
-        track('sendFailure');
+      const onError = (err?: Error): void => {
+        const batchInfo = credits?.find(batch => batch.denom === batchDenom);
```

Comment:
I believe you could just extract this outside `onError` and `onSuccess` and use `const batchInfo = credits[creditSendOpen];` so you don't have to use `find()` multiple times.